packages/notebook/src/notebooktools.ts

```diff
@@ -38,6 +38,17 @@ import { NotebookPanel } from './panel';
 import { INotebookModel } from './model';
 import { INotebookTools, INotebookTracker } from './tokens';
 
+/**
+ * A type alias for a readonly partial JSON tuples `[option, value]`.
+ * `option` should be localized.
+ *
+ * Note: Partial here means that JSON object attributes can be `undefined`.
+ */
+export declare type ReadonlyPartialJSONOptionValueArray = [
+  ReadonlyPartialJSONValue | undefined,
+  ReadonlyPartialJSONValue
+][];
+
```

Comment:
Where should this go @blink1073?

Also `export type` instead of  `export declare type` ?

Reply:
It should be exported as `export type` in the `NotebookTools` namespace.

Reply:
Fixed!